src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -652,7 +662,8 @@ public Long getValue()
 
         readRepairRequests = createTableMeter("ReadRepairRequests");
         shortReadProtectionRequests = createTableMeter("ShortReadProtectionRequests");
-        replicaSideFilteringProtectionRequests = createTableMeter("ReplicaSideFilteringProtectionRequests");
+        replicaFilteringProtectionRequests = createTableMeter("ReplicaFilteringProtectionRequests");
```

Comment:
I think we never documented this metric, so I guess it's ok to change the name of the metric even though we have releases with the old name (3.0.21 and 3.11.7). I think that for 3.11 and trunk PRs we should add `ReplicaFilteringProtectionRequests` and `ReplicaFilteringProtectionRowsCachedPerQuery` to [`metrics.rst`](https://github.com/apache/cassandra/blob/cassandra-3.11/doc/source/operating/metrics.rst#table-metrics).

Reply:
Thanks for the reminder.